pkg/apis/application/v1alpha1/applicationset_types.go

```diff
@@ -804,7 +804,8 @@ type ApplicationSetStatus struct {
 	Conditions        []ApplicationSetCondition         `json:"conditions,omitempty" protobuf:"bytes,1,name=conditions"`
 	ApplicationStatus []ApplicationSetApplicationStatus `json:"applicationStatus,omitempty" protobuf:"bytes,2,name=applicationStatus"`
 	// Resources is a list of Applications resources managed by this application set.
-	Resources []ResourceStatus `json:"resources,omitempty" protobuf:"bytes,3,opt,name=resources"`
+	Resources      []ResourceStatus `json:"resources,omitempty" protobuf:"bytes,3,opt,name=resources"`
+	ResourcesCount int64            `json:"resourcesCount,omitempty" protobuf:"varint,4,opt,name=resourcesCount"`
```

Comment:
Should have a docstring for this, particularly to explain it may differ from the actual length of resources.

Reply:
Makes sense. Added documentation